alphab/alphab.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
+import matplotlib.dates as mdates
+
+from scipy.stats import shapiro
+from scipy.stats import ttest_ind
+from scipy.stats import f
+from scipy.stats import mannwhitneyu
+from pathlib import Path
+
+
+class BucketTest:
+    """ BucketTest class computes and renders charts and statistics for bucket testing """
+
+    def __init__(self, df: pd.DataFrame, variable: str, group: str, x_axis='date', custom_title="", custom_day_interval=1,
+                 custom_ylabel=""):
+        """ Create a new bucket test with the given attributes """
+        self.df = df
+        self.variable = variable
+        self.x_axis = x_axis
+        self.group = group
+        self.custom_title = custom_title
+        self.custom_day_interval = custom_day_interval
+        self.custom_ylabel = custom_ylabel
+
+    def render(self, figure_size_x=12, figure_size_y=5, line_width=3, title_font_size=16, legend_font_size=14):
+        """ Render renders the charts representing the bucket test """
+
+        fig, ax = plt.subplots(figsize=(figure_size_x, figure_size_y))
+        for group_value in self.df[self.group].unique():
+            df = self.df[self.df[self.group] == group_value]
+            df.set_index(self.x_axis, drop=False, inplace=True)
+            ax.plot(df[self.variable], label=group_value, linewidth=line_width)
+
+        # Title customization
+        if self.custom_title != "":
+            plt.title(self.custom_title, fontsize=title_font_size)
+        else:
+            plt.title('{} per {}'.format(self.variable, self.group), fontsize=title_font_size)
+        plt.legend(bbox_to_anchor=(1.3, 0.8), frameon=False, fontsize=legend_font_size)
+
+        # Y-label customization
+        if self.custom_ylabel != "":
+            plt.ylabel(self.custom_ylabel)
+        else:
+            plt.ylabel(self.variable)
```

Comment:
```suggestion
            plt.ylabel(self.custom_ylabel or self.variable)
```
^ Should work and it's shorter